hq/app/aws/support/TrustedAdvisor.scala

```diff
@@ -11,10 +11,13 @@ import logic.DateUtils.fromISOString
 import utils.attempt.Attempt
 
 import scala.collection.JavaConverters._
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.ExecutionContext
 
 
 object TrustedAdvisor {
+  val portPriorityMap = Map("27017" -> 0, "27018" -> 0, "27019" -> 0, "6379" -> 0, "6380" -> 0,  "9000" -> 1, "7077" -> 1, "4040" -> 1, "8890" -> 1, "4041" -> 1, "4042" -> 1, "8080" -> 1, "22" -> 9)
```

Comment:
`"22" -> 9`

Why is 22 getting assigned 9 here? Should it be 2?

Reply:
chasing it to 9 in the test

Reply:
Would it be easier to just use the list index as the priority and make a decision about the relative importances of each port number? That will also help provide a more stable sort across accounts.

Reply:
Some examples that appear to be missing are:

5432 (postgres)
5601 (kibana)
9300, 9200 (elastic search)
0-65535 (wide open)

Have a look at the existing results to find ports that are are flagged now. If any of those are important ones (especially databases) then make sure this prioritisation captures them.

Reply:
didn't add postgress, mysql, etc, as they're already flagged as Red. 
Wide open is also flagged as Red.

Reply:
Sure, but since we're already doing a sort we could make it stable (consistent across changing data and different accounts) by implementing a sort that's 'stable' across the various ports.

Reply:
not sure about deriving priority from the index. More importantly, I think wide port security group rule will break it.